src/arrays/multidimensional_arrays/hard/q4/AlterTable.java

```diff
@@ -0,0 +1,22 @@
+package arrays.multidimensional_arrays.hard.q4;
+
+/*
+Make a program that defines a 2D array and goes through each element in it by traversing each row's elements one at a
```

Comment:
Change question text to:
"Make a program that defines a multidimensional array and iterates over each row of the array, printing out every other element starting from the first element of the first row. For example, if you have the following 2D array {{1,2,3}, {4, 5 ,6}, {7,8,9} }, the program will print 1, 3, 5, 7, 9."

Reply:
The question text was not changed